### PR TITLE
Set up CI

### PR DESCRIPTION
**Pull request by gkucsko**

### CODE REVIEW

**Review by poneill — APPROVED** [review]
lgtm

File: pyctcdecode/decoder.py [hunk 1]
```diff
@@ -77,7 +87,7 @@ def _log_softmax(x: np.ndarray, axis: Optional[int] = None) -> np.ndarray:
     exp_tmp = np.exp(tmp)
     # suppress warnings about log of zero
```
Comment (poneill) [inline pyctcdecode/decoder.py:88]: would it be better just to add epsilon=10**-100 to all values before taking logs?

**Review by poneill — APPROVED** [review]
would just try to make the noqa exceptions as tight as possible so we don't leave room for bugs to hide

File: pyctcdecode/alphabet.py [hunk 2]
```diff
@@ -38,14 +38,16 @@ def _convert_bpe_format(token: str) -> str:
         return token[2:]
     elif token == BPE_CHAR:
         return token
-    elif token == "":
+    elif token == "":  # nosec
```
[the remark this answers is not in the document]
Comment (gkucsko) [reply pyctcdecode/alphabet.py:41]: it thinks this might be a hardcoded "token"

File: pyctcdecode/decoder.py (hunk 1 above)
Comment (gkucsko) [reply pyctcdecode/decoder.py:88]: idk, this is taken straight from scipy

**Review by poneill — APPROVED** [review]

**Review by poneill — APPROVED** [review]

**Pull request merged**